projects/ion/src/lib/chip/chip.component.html

```diff
@@ -0,0 +1,65 @@
+<button
+  data-testid="ion-chip"
+  [id]="chipId"
+  [disabled]="disabled()"
+  [class]="'ion-chip chip-' + size()"
+  [ngClass]="{
+    chip: !selected(),
+    'chip-selected': selected(),
+  }"
+  ionDropdown
+  [dropdownConfig]="{
+    shouldRender: !disabled() && hasDropdown(),
+    required: required(),
+    multiple: multiple(),
+    clearButton: multiple(),
+  }"
+  [dropdownOptions]="options() ? options() : []"
+  (dropdownOpened)="select()"
+  (dropdownEvent)="selectDropdownItem($event)">
+  @if (
+    (!hasDropdown() && icon()) ||
+    (hasDropdown() &&
+      dropdownWithIcon &&
+      iconPlaceholder &&
+      iconPosition() === 'left')
+  ) {
```

Comment:
Acredito que essas verificações ficariam melhor em uma função dentro do TS.

```suggestion
  @if (shouldShowIcon) {
```